### PR TITLE
WIP: Initial refactor, Windows dev env instructions

### PR DESCRIPTION
**Pull request by myhndl**

### CODE REVIEW

File: _i18n/en/_pages/dev_env.md [hunk 1]
```diff
@@ -0,0 +1,149 @@
+# Developing for PolySwarm
+
+Developers interested in building (BUIDLing) for the PolySwarm marketplace should start here!
+
+PolySwarm supports both Linux and Windows based engines (arbiters, ambassadors & microengines).
+
+Our Linux-based engines are fully Dockerized and our Windows-based engines are fully virtualized; the instructions presented here reflect these recommendations, but of course feel free to bundle your engine howevery you like.
```
Comment (mrtizmoatwork) [inline _i18n/en/_pages/dev_env.md]: spelling error "howevery" -> "however"

File: _i18n/en/_pages/dev_env.md [hunk 2]
```diff
@@ -0,0 +1,149 @@
...
+### Grab `polyswarm-client`
+
+The easiest way to get going on building a PolySwarm engine is to build on our `polyswarm-client`.
+
+We're done with the elevated PowerShell - close this PowerShell and open a new (not elevated) PowerShell.
+
+Run the following command in the non-privileged PowerShell:
+
+```
+git clone https://github.com/polyswarm/polyswarm-client.git
```
Comment (mrtizmoatwork) [inline _i18n/en/_pages/dev_env.md]: what do you think about changing this to be something like:

`pip install git+https://github.com/polyswarm/polyswarm-client.git#egg=polyswarm-client`

That will install polyswarm-client and all dependencies.

Comment (mrtizmoatwork) [reply _i18n/en/_pages/dev_env.md]: The, once we have polyswarm-client in pypi, we can just change the instruction to be:

`pip install polyswarm-client`

Comment (mrtizmoatwork) [reply _i18n/en/_pages/dev_env.md]: OR, is it your intention that a dev may want to modify the polyswarm-client code and repeatedly install their own variants from the local codebase?

File: _i18n/en/_pages/dev_env.md (hunk 1 above)
Comment (myhndl) [reply _i18n/en/_pages/dev_env.md]: thanks

File: _i18n/en/_pages/dev_env.md (hunk 2 above)
Comment (myhndl) [reply _i18n/en/_pages/dev_env.md]: That wasn't my intention, your way of doing it is a lot simpler and cleaner, thanks. Will change.

**Comment by myhndl** [comment]
Closing this PR, now tracking in `feature/overhaul`

**Pull request closed without merging**